cms/static/js/views/active_video_upload_list.js

```diff
@@ -101,7 +117,18 @@ define(
             // indicate that the correct upload url has already been retrieved
             fileUploadAdd: function(event, uploadData) {
                 var view = this,
-                    model;
+                    model,
+                    errorMsg;
+
+                // Validate file
+                errorMsg = view.validateFile(uploadData);
+                if (errorMsg) {
+                    view.showErrorMessage(errorMsg);
+                    return;
```

Comment:
We can avoid this return if we move rest of the code in this function in else part.